install.sh

```diff
@@ -78,12 +78,12 @@ if ! $packaging; then
     has_java=false
     if [ -x /usr/bin/java ]; then
         javaver=$(/usr/bin/java -version 2>&1|head -n1|cut -f 3 -d " ")
-        if [[ "$javaver" =~ ^\"1.8.0 ]]; then
+        if [[ "$javaver" =~ ^\"1.8.0 || "$javaver" =~ ^\"11.0.8 ]]; then
```

Comment:
Should we relax the version check a bit? IOW, lose that patch level (".8") to avoid updating this all the time as new Java versions are released?

Reply:
I considered that, but 1.8.0 is fixed version. Not sure the major version will be change.

However, in https://packages.debian.org/search?keywords=openjdk-11-jre-headless
I saw both 11.0.6 and 11.0.8

Let's change it to 11.0